### ISSUE TITLE
Retry import (with exponential backoff) if an error occurs

### ISSUE THREAD

**Issue by maxammann** [digitalfabrik/entitlementcard#528]
**Is your feature request related to a problem? Please describe.**
We experienced an error while importing new data. Once because of a connection reset and once because of malformatted XML.

```
[main] INFO app.ehrenamtskarte.backend.stores.importer.LbeDataImporter - Unknown exception while downloading data from lbe
com.fasterxml.jackson.databind.JsonMappingException: Unexpected EOF; was expecting a close tag for element <anbieter>
```
```
[main] INFO app.ehrenamtskarte.backend.stores.importer.LbeDataImporter - Unknown exception while downloading data from lbe
java.io.IOException: Connection reset by peer
```

**Describe the solution you'd like**
This was an error which happened only once. On the day after the failure it worked again. Mabye we should retry if the import fails.

**Comment by klinzo** [comment]
Would it be possible to save a snapshot of the db before the import and to rollback if it fails?

**Comment by Schedulaar** [comment]
A normal DB transaction should do the trick, right?

**Comment by klinzo** [comment]
Oh yeah right, we are deleting the stores and importing it in the same transaction so we should be fine. I somehow thought we do that in two separate places.

**Comment by sarahsporck** [comment]
@michael-markl has already done that :)
